app/models/record.server.ts

```diff
@@ -0,0 +1,98 @@
+import type { Record, User } from '@prisma/client';
+import { prisma } from '../db.server';
+
+export const createRecordInDB = async (
+  username: User['username'],
+  type: Record['type'],
+  name: Record['name'],
+  value: Record['value'],
+  createdAt: Record['createdAt'],
```

Comment:
`createdAt` and `updatedAt` should be handled internally by Prisma.  You shouldn't pass them in or set them manually.  Same applies throughout this PR (please look for other places with the same problem and fix).

Reply:
Should have discussed. Will be all updated.